filebeat/docs/inputs/input-filestream-file-options.asciidoc

```diff
@@ -117,6 +141,36 @@ If a file that's currently being harvested falls under `ignore_older`, the
 harvester will first finish reading the file and close it after
 `close.on_state_change.inactive` is reached. Then, after that, the file will be ignored.
 
+[float]
+[id="{beatname_lc}-input-{type}-ignore-inactive"]
+===== `ignore_inactive`
+
+If this option is enabled, {beatname_uc} ignores every file that has not been
+updated since the selected time. Possible options are `since_first_start` and
+`since_last_start`. The first option ignores every files that has not been updated since
+the first start of {beatname_uc}. It is useful when the Beat might be restarted
+due to configuration changes or due to a failure. The second option is the same
+as the `tail_files` option of `log` input, it reads from files that has been updated
+since the start of {beatname_uc}.
```

Comment:
> The second option is the same as the `tail_files` option of `log` input, it reads from files that has been updated since the start of {beatname_uc}.

Although true, we should not mention `tail_files` or `log` input. Lets keep the documentation here self contained.